nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -49,6 +49,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `himalaya` was updated to v1.0.0-beta, which introduces breaking changes. Check out the [release note](https://github.com/soywod/himalaya/releases/tag/v1.0.0-beta) for details.
 
+- `dockerTools.streamLayeredImage` created layers with mtimes using the "now" argument for the `created` parameter. It now uses UNIX time of "1" for the mtime of layers. It will still use "now" for the last customization layer and for setting the creation date of the image itself. This leads to greater re-use of layers between different images.
```

Comment:
```suggestion
- `dockerTools.streamLayeredImage` previously created layers with mtimes using the "now" argument for the `created` parameter. It now uses UNIX time of "1" for the mtime of layers. It will still use "now" for the last customization layer and for setting the creation date of the image itself. This leads to greater re-use of layers between different images.
```